dependencies.yaml

```diff
@@ -8,7 +8,6 @@ files:
     includes:
       - cudatoolkit
       - develop
-      - doc
```

Comment:
GH won't let me add a code suggestion to deleted lines, but can we add this entry back with the new `docs` name?

The `files.all` entry is intended to include all of the children dependency lists, `docs` included.

Reply:
ah got it, added it back